### PR TITLE
Telegram: forcebuy inline keyboard

### PR DESCRIPTION
**Pull request by gmatheu**
## Summary
Inline keyboard for **forcebuy** command

Solve the issue: n/a

## Quick changelog

- Added inline keyboard response 

## What's new?

This PR adds a functionality meant to facilitate forcebuys. When you execute **/forcebuy** without arguments, it generates an inline keyboard with the allow-listed pairs. If you click any of those buttons it executes the forcebuy for that pair.

Screen Shot 2021-04-15 at 21 48 02

### CODE REVIEW

File: freqtrade/rpc/telegram.py [hunk 1]
```diff
@@ -610,6 +616,25 @@ def _forcesell(self, update: Update, context: CallbackContext) -> None:
         except RPCException as e:
             self._send_msg(str(e))
 
+    def _forcebuy_action(self, pair, price=None):
+        try:
+            self._rpc._rpc_forcebuy(pair, price)
+        except RPCException as e:
+            self._send_msg(str(e))
+
+    def _forcebuy_inline(self, update: Update, _: CallbackContext) -> None:
```
Comment (xmatthias) [inline freqtrade/rpc/telegram.py:652]: I'm not entirely sure about this (i never used inline-keyboards with telegram so far) - but would we need to add the `@authorize_only` tag to this method too, or is this only reachable as part of an answer?

**Review by xmatthias — APPROVED** [review]
Overall looks nice (although i didn't play with it yet) ... 

Is there a way to "abort" the forcebuy? 
say i sent a `/forcebuy` by mistake - and now i don't really want to do this anymore? can i simply answer with something else, or should we add a "abort" button to the inline answer for easy abortion?


*edit* was meant to be a comment, not an "approve" ... not yet, anyway :laughing:

**Comment by xmatthias** [comment]
@gmatheu you submitted this as "WIP" / draft ... 
however this seems pretty complete ... would you mind letting me know what you think is still missing?

File: freqtrade/rpc/telegram.py (hunk 1 above)
Comment (gmatheu) [reply freqtrade/rpc/telegram.py:652]: Let me double-check that...

Comment (xmatthias) [reply freqtrade/rpc/telegram.py:652]: Trying this, i found no way to get to this callback without having an authorization first - so all fine.

**Review by xmatthias — APPROVED** [review]
LGTM :+1: 
Great enhancement!

**Pull request merged**